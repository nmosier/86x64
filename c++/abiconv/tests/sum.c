#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
   for (int i = 0; i < argc; ++i) {
      printf("%s %s\n", argv[i], argv[i]);
   }
   return 0;
}
