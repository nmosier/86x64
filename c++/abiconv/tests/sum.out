./sum64 ./sum64
